### PR TITLE
Feat: Made the pages responsive

### PR DESCRIPTION
**Pull request by LordUttam**
Made the pages responsive and added filters menu that opens with floating action button in mobile viewports. Please review thanks :) 
image
image

### CODE REVIEW

**Review by Mohasin-Haque — COMMENTED** [review]
Everything looks good to me.. You may go ahead

**Review by aurobindoGupta — APPROVED** [review]
nice

File: pages/cart.html [hunk 1]
```diff
@@ -35,67 +35,71 @@
...
             <h2 class="h--2 cart__heading m__t-1 m__b-0">My Cart</h2>
 
-            <div class="card card--horizontal">
-                <div class="card__image">
-                    <img src="../assets/auto-yogi.jpg">
-                </div>
+            <div class="card__container justify--center items--center m--y-1">
+                <div class="card card--horizontal">
+                    <div class="card__image flex justify--center">
+                        <img src=" ../assets/auto-yogi.jpg">
```
Comment (sk-ip) [inline pages/cart.html:46]: you may add a meaningful alt tag

Comment (LordUttam) [reply pages/cart.html:46]: nice catch I missed this :) thank you.

**Pull request merged**